docs/interest-filtering.md

```diff
@@ -0,0 +1,105 @@
+# Interest filters
+
+## `pantheon.ei.localized_terms`
+
+Modify terms before they are localized.
+
+### Parameters
+
+$terms __(array)__ An array of terms to modify.
+
+### Example
+
+Add term to array:
+```bash
```

Comment:
All of these code snippets are PHP, so the language after the backticks should be PHP for proper code styles (not sure how/if this applies to the official documentation). 🔢

```suggestion
```php
```